packages/fether-react/src/stores/healthStore.js

```diff
@@ -25,20 +29,27 @@ export const STATUS = {
 export class HealthStore {
   @observable nodeHealth;
   @observable syncing;
+  @observable clockSync;
 
-  constructor () {
+  constructor() {
     nodeHealth$().subscribe(this.setNodeHealth);
     syncing$().subscribe(this.setSyncing);
+
+    const { ipcRenderer } = electron;
```

Comment:
Just tried on browser, it throws an error.

Maybe add something like in parityStore? https://github.com/paritytech/fether/blob/master/packages/fether-react/src/stores/parityStore.js#L34-L39